src/api/routes/group.ts

```diff
@@ -0,0 +1,74 @@
+import { Router } from "express";
+import { Container } from "typedi";
+import GroupService from "../../services/GroupService";
+
+const groupRouter = Router();
+const groupService = Container.get(GroupService);
+
+const handler = async (cb: () => Promise<void>, res) => {
```

Comment:
callback approach is not really nice. It's better to use hof. Example can be found here https://github.com/Abazhenov/express-async-handler/blob/master/index.js

Reply:
Done.